src/main/kotlin/no/nb/mlt/wls/infrastructure/callbacks/NotificationOrderPayload.kt

```diff
@@ -59,9 +65,13 @@ data class NotificationOrderPayload(
     )
     val owner: Owner?,
     @Schema(
-        description = "Who's the receiver of the material in the order."
+        description = "The person to contact regarding matters about the order"
     )
-    val receiver: Order.Receiver,
+    val contactPerson: String,
+    @Schema(
+        description = "The address of the receiver of the material in the order."
+    )
+    val address: Order.Address?,
```

Comment:
Notice: note not noted in notification, not nice, no, no... Note the note in notification now.